### ISSUE TITLE
No AWS FSx Lustre PERSISTENT_2 data association parameter

### ISSUE THREAD

**Issue by olileach** [hashicorp/terraform-provider-aws#22656]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version

➜ terraform terraform -v
Terraform v1.1.2
on darwin_arm64
provider registry.terraform.io/hashicorp/aws v3.72.0

### Affected Resource(s)

- aws_fsx_lustre_file_system

### Terraform Configuration Files

```
terraform {
  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = "3.72.0"
    }
  }
}

# Configure the AWS Provider
provider "aws" {
  region = "us-east-1"
}

resource "aws_fsx_lustre_file_system" "fsx-filesystem-1" {
  import_path      = "s3://bucket-us-east-1"
  storage_capacity = 1200
  storage_type = "SSD"
  per_unit_storage_throughput = "125"
  deployment_type = "PERSISTENT_2"
  subnet_ids       = ["subnet-xxxxx"]
}
```
### Debug Output

```
aws_fsx_lustre_file_system.fsx-filesystem-1: Creating...
╷
│ Error: error creating FSx Lustre File System: BadRequest: Linking a Persistent 2 file system to an S3 bucket using the LustreConfiguraton is not supported. Create a file system and then create a data repository association to link S3 buckets to the file system. For more details, visit https://docs.aws.amazon.com/fsx/latest/LustreGuide/create-dra-linked-data-repo.html
│
│   with aws_fsx_lustre_file_system.fsx-filesystem-1,
│   on fsx.tf line 15, in resource "aws_fsx_lustre_file_system" "fsx-filesystem-1":
│   15: resource "aws_fsx_lustre_file_system" "fsx-filesystem-1" {
│
╵
```

### Expected Behavior

I am trying to deploy an AWS FSX PERSISTENT_2 file system. PERSISTENT_2 was released in Nov 2021 and is built on Lustre v2.12 and offers higher PerUnitStorageThroughput. 

However, I need to create a data repository association with the PERSISTENT_2 filesystem but can't using the v3.72.0 AWS provider. The import_path parameter is not compatible with PERSISTENT_2 and there are no other parameters I can use to create the data repository association.

### Actual Behavior

The deployment fails if I use import_path using PERSISTENT_2. If I remove the import_path parameter, I get a PERSISTENT_2 filesystem that uses Lustre version 2.12. However, there is no way, using the AWS provider v3.72.0, to create a data reposiotry association so I cannot import files to FSx from S3.

### Steps to Reproduce

1. Take the example tf file above that include the import_path parameter
2. terraform plan | apply
3. The deployment will fail with the error above
4. remove the import_path
5. The deployment will work and deploy a lustre 2.12 filesystem but without the data repository association

### References

https://aws.amazon.com/about-aws/whats-new/2021/11/amazon-fsx-lustre-file-systems/

**Comment by DrFaust92** [comment]
https://github.com/hashicorp/terraform-provider-aws/pull/22291 Related

**Comment by breathingdust** [comment]
Closed in `v3.73.0` 🎉
